src/py42/sdk/queries/fileevents/v2/filters/destination_filter.py

```diff
@@ -0,0 +1,102 @@
+from py42.choices import Choices
+from py42.sdk.queries.fileevents.file_event_query import FileEventFilterStringField
+from py42.sdk.queries.query_filter import QueryFilterStringField
+
+
+class Destination(QueryFilterStringField):
```

Comment:
Can we distinguish the version in the docstring for each filter class? Since they have the same class name, it could help catch import bugs if REPL/IDE auto-complete could show you what version it was.

Maybe `"""V2 Filter Class that filters events based on the destination name."""` ?